lib/middleware/map-request-params.js

```diff
@@ -70,13 +70,13 @@ function parseKeyword(req, res) {
 }
 
 function parseCampaignId(req) {
+  let campaign = {};
   if (req.broadcast_id && req.broadcastContentfulObject) {
-    const broadcastCampaign = getCampaignFromContentfulObject(req.broadcastContentfulObject);
-    req.campaignId = broadcastCampaign.campaignId;
+    campaign = getCampaignFromContentfulObject(req.broadcastContentfulObject);
   } else if (req.keyword && req.keywordContentfulObject) {
-    const keywordCampaign = getCampaignFromContentfulObject(req.keywordContentfulObject);
-    req.campaignId = keywordCampaign.campaignId;
+    campaign = getCampaignFromContentfulObject(req.keywordContentfulObject);
```

Comment:
This function name feels a little vague, since we have different Phoenix and Contentful Campaigns objects. What do you think about renaming to something like `getContentfulCampaignFromObject`. `getContentfulCampaignFromContentfulObject` is awkward for sure (but I can see the advantage of keeping the `ContentfulObject` naming since you've defined it that way in other places).